easy_lausanne/maskcreation.py

```diff
@@ -189,6 +189,7 @@ def create_roi(subject_id):
     fs_label_dir = op.join(fs_dir, 'label')
     # load aseg volume
     aseg = ni.load(op.join(fs_dir, 'mri', 'aseg.nii.gz'))
+	asegdims = aseg.shape
```

Comment:
Does this not cause an indentation error?

Reply:
I was also wondering about this, but oddly, the script works just fine. In my text editor, these lines appear aligned. I'll try with another editor and see if I can fix it so it both works and looks aligned in git

Reply:
It looks like I was using a tab instead of 4 spaces for the indentation (or at least that's what it looks like in Notepadd++), which should have caused an error, but somehow didn't.
It's now fixed